### ISSUE TITLE
Can't place markers in 2.9.6+1.20

### ISSUE THREAD

**Issue by murphy-slaw** [sleepingdragoninn/antique-atlas#157]
Fabric Loader 0.15.11
antique-atlas 2.9.6+1.20
fabric-api 0.83.0+1.20.1

Placing markers on the map makes the scribbling sound but no marker appears on the  map. At first I thought the problem was that markers without names weren't working, but named markers do the same thing.

Curiously, surveyor seems to think that *something* exists, but it's confused:

```
[23:06:54] [Render thread/INFO]: [System] [CHAT] [Surveyor] ---Landmark Types---
[23:06:54] [Render thread/INFO]: [System] [CHAT] surveyor:point: created 0 {of 3}
[23:06:54] [Render thread/INFO]: [System] [CHAT] [Surveyor] -------End Landmarks------
```

and after trying to add another marker:

```
[23:08:07] [Render thread/INFO]: [System] [CHAT] [Surveyor] ---Landmark Types---
[23:08:07] [Render thread/INFO]: [System] [CHAT] surveyor:point: created 0 {of 4}
[23:08:07] [Render thread/INFO]: [System] [CHAT] [Surveyor] -------End Landmarks-------
```

[latest.log](https://github.com/user-attachments/files/16115044/latest.log)

**Comment by sleepingdragoninn** [comment]
lmao yeah i know exactly why that is I'll get right on that.

client UUID helper was not adjusted to match the new logic for the server UUID helper in singleplayer.

**Comment by sleepingdragoninn** [comment]
should be fixed in 2.6.7 - thanks!
